src/couch_mrview/src/couch_mrview_debug.erl

```diff
@@ -34,7 +50,100 @@ help(view_signature) ->
     io:format("
     view_signature(ShardName, DDocName)
     --------------
+
     Returns a view signature for given ddoc for a given (non clustered) database.
+
+    ---
+    ", []);
+help(index_state) ->
+    io:format("
+    index_state(Pid)
+    --------------
+
+    Pid: Pid of couch_index:init/1, specifically an mrview index.
+
+    Returns a state map that includes the signature, db_name, idx_name, update_seq,
```

Comment:
The inline representation of fields list is hard to read. Can you use something like:

```
Returns a state map that includes following fields:

  - db_name
  - idx_name
  - ...
```

Also update documentation for other functions bellow.